fv3core/fv3core/utils/gt4py_utils.py

```diff
@@ -115,13 +115,6 @@ def make_storage_data(
             else:
                 mask = (n_dims * (True,)) + ((max_dim - n_dims) * (False,))
 
-    if n_dims == 1:
```

Comment:
I expect this breaks the GPU tests. If it does not then that's cool, otherwise I'll have to propagate the backend argument into these helper routines.

Reply:
So this does break the tests but not in the way I expected, will revert and propagate the argument.

What this function does in terms of broadcasting dimensions is pretty complex. We might want to avoid using it when we want to do a simple conversion of a numpy array of a given shape into a storage of an identical shape, and use it mainly for converting Fortran data from savepoints into storages of the right shape.